src/_guides/openapi/specification/v3.1/data-models/schema-and-data-types.md

```diff
@@ -281,6 +281,42 @@ This schema mandates that the properties "name" and "age" must be present in the
 
 For more information on JSON Schema validation keywords, you can refer to the [JSON Schema Validation documentation](https://json-schema.org/learn/validation.html).
 
+## readOnly & writeOnly
+
+JSON Schema provides `readOnly` and `writeOnly` boolean keywords, which are really helpful in the context of an API, because resources usually available in two flavours: the representation of a resource in a request body, and the representation of the resource in a response body.
```

Comment:
```suggestion
JSON Schema provides `readOnly` and `writeOnly` boolean keywords, which are really helpful in the context of an API, because resources are usually available in two flavours: the representation of a resource in a request body, and the representation of the resource in a response body.
```